app/backend/api/internal/persistence/repository_integration_test.go

```
@@ -0,0 +1,325 @@
+}
+
+func tableColumns(t *testing.T, db *sql.DB, table string) map[string]string {
+	t.Helper()
+	rows, err := db.Query(`
+SELECT column_name, data_type
+FROM information_schema.columns
+WHERE table_schema = current_schema() AND table_name = $1`, table)
+	if err != nil {
+		t.Fatalf("columns for %s: %v", table, err)
+	}
+	defer rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)